### PR TITLE
added the lightning_flash_accumulation_in_vicinity cell method missing from imengluk

### PR DESCRIPTION
**Pull request by maxwhitemet**
Addresses [#802 ](https://github.com/metoppv/mo-blue-team/issues/802)

Testing:
 - [x] Ran tests and they passed OK
 - [x] Added new tests for the new feature(s)

### CODE REVIEW

**Review by lambert-p — COMMENTED** [review]
I have reviewed your code, it is looking good and I can't see any changes that need to be made. I will put it into second review.

**Review by brhooper — CHANGES_REQUESTED** [review]
Thanks @maxwhitemet, these changes look as I expected them to. 

I've attempted to run all of the IMPROVER tests and get failures for acceptance tests in `improver_tests/acceptance/test_lightning_from_cape_and_precip.py`. I think that the KGOs for these tests need updating to include the new cell method.

**Review by brhooper — APPROVED** [review]
Thanks @maxwhitemet, this all looks fine to me.

I think you'll need to pull master back into this branch before it can be merged, so that the fix to the Github actions can be included.

**Review by lambert-p — APPROVED** [review]

**Pull request merged**
